src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -101,7 +101,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
     <CheckForOverflowUnderflow Condition="'$(CheckForOverflowUnderflow)'==''">false</CheckForOverflowUnderflow>
     <AutomaticallyUseReferenceAssemblyPackages Condition="'$(AutomaticallyUseReferenceAssemblyPackages)'==''">true</AutomaticallyUseReferenceAssemblyPackages>
-    <MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>1.0.0</MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>
+    <MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>1.0.1</MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>
```

Comment:
This version has a bug and @NikolaMilosavljevic is working on publishing a fix.

Reply:
Can you share a link to the bug? I'm very curious.

Reply:
https://github.com/microsoft/dotnet/issues/1137#issuecomment-833107242

Reply:
https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/1.0.2 is now published.